client/packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -22,13 +32,20 @@ import { useOutbound } from '../api';
 import { AppRoute } from '@openmsupply-client/config';
 import { OutboundLineFragment } from '../api/operations.generated';
 
+const TAB_HEIGHT = '32px';
```

Comment:
trying to make the tabs take up as little screen space as possible. The default is 42px and it isn't easily changed. Would like to lift out as much of this as possible before applying to the other areas.